WordPress/Classes/ViewRelated/Me/AccountCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AccountCell.swift
+//  WordPress
+//
+//  Created by Gonzalo G Erro on 11/10/16.
+//  Copyright © 2016 WordPress. All rights reserved.
+//
+
```

Comment:
I thought about removing it because I saw it, but as I didn't remember reading about it in the coding style doc I wasn't sure it was official (my bad I should had asked not assume) so removing it